src/app/page.tsx

```diff
@@ -65,45 +65,3 @@ export default function Home() {
         </Stack>
     )
 }
-
-const Approve: FC<{ fileName: string }> = ({ fileName }) => {
-    const approve = useApproveRun()
-    return <Button onClick={() => approve.mutate(fileName)}>Approve</Button>
-}
-
-const Results: FC<{ fileName: string; records: CSVRecord[] }> = ({ fileName, records }) => {
-    const [opened, { open, close }] = useDisclosure(false)
-    const columns = Object.keys(records[0]).map((key: string) => ({
-        accessor: key,
-        title: key,
-    }))
-
-    return (
-        <>
-            <Modal
-                opened={opened}
-                onClose={close}
-                size="100%"
-                title={`Results for ${fileName}`}
-                scrollAreaComponent={ScrollArea.Autosize}
-                overlayProps={{
-                    backgroundOpacity: 0.55,
-                    blur: 3,
-                }}
-                centered
-            >
-                <DataTable
-                    idAccessor={columns[0].title}
-                    withTableBorder={false}
-                    withColumnBorders={true}
-                    records={records}
-                    columns={columns}
-                />
-            </Modal>
-
-            <Button variant="outline" onClick={open}>
-                View Results
-            </Button>
-        </>
-    )
-}
```

Comment:
Moved to their own components because nextjs is not a fan of exporting these from the page.tsx file 😡